src/core/lib/iomgr/socket_utils_posix.h

```diff
@@ -31,10 +31,25 @@
 #include "src/core/lib/iomgr/socket_factory_posix.h"
 #include "src/core/lib/iomgr/socket_mutator.h"
 
+#ifdef GRPC_LINUX_ERRQUEUE
+
```

Comment:
nit: please remove the new lines between the macro defs.

Reply:
Done.